### DISCUSSION TITLE
deactivating multihops / finding the direct pair of a token

### DISCUSSION THREAD

**Discussion by vaidab** [uniswap-python/uniswap-python#223] [Q&A]
When a token is listed on pancakeswap (liquidity is added), the token has a direct pair (TOKEN/USDT). Trading TOKEN/BNB early will have a very bad exchange rate.
How can I find the direct pair of the token or disable "multihops" and get an error if it's not a correct pair?

[Here's](https://stackoverflow.com/questions/70409395/swapexacttokensfortokens-transaction-has-different-ratio-than-router-interface-t) my specific stackoverflow question that brought me here.

**Comment by romanbsd** [comment]
Similar to https://github.com/uniswap-python/uniswap-python/issues/205
